vxvas2nets/vas2nets_sms.py

```diff
@@ -23,6 +23,10 @@ class Vas2NetsSmsTransportConfig(HttpRpcTransport.CONFIG_CLASS):
         "null for no timeout",
         default=None)
 
+    reply_outbound_url = ConfigText(
+        "Url to use for reply outbound messages",
+        required=True)
```

Comment:
Just checking that `required=True` is intentional here. I like the idea of asking people to explicitly use `reply_outbound_url: null`, but wasn't sure if that was what was intended.